ecobee/beam_batch.py

```diff
@@ -0,0 +1,144 @@
+import argparse
```

Comment:
In general, I would do it by writing some kind of interface that is reusable and then call that interface to get the stuff done.
I usually write a utility class for one-of functions. And then there would be one file per-class, just like in Java. In that file you would have a class that logically groups some methods together. You can then use a Dependency injection to pass along different objects.

Reply:
I did not create a utility file because I think it would be extra for the purpose of this exercise. But let me know if you think otherwise.